src/test/java/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,153 @@
+package subway.acceptance;
+
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.acceptance.LineRequestFixture.*;
+import static subway.acceptance.StationRequestFixture.지하철_역_생성;
+
+@DisplayName("지하철 노선 관련 기능")
+public class LineAcceptanceTest extends AcceptanceTest {
+
+    /**
+     * Given 2개의 지하철 역을 생성하고
+     * When (해당 지하철 역을 포함하는) 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선을 생성 한다.")
+    @Test
+    void createLine() {
+        // given
+        Long upStationId = 지하철_역_생성("강남역").jsonPath().getLong("id");
+        Long downStationId = 지하철_역_생성("양재역").jsonPath().getLong("id");
+
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_생성("신분당선", "bg-red-600", upStationId, downStationId, 10L);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // then
+        List<String> lineNames = 지하철_노선_목록_조회().jsonPath().getList("name", String.class);
+        assertThat(lineNames).containsAnyOf("신분당선");
+    }
+
+
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     */
+    @DisplayName("지하철 노선을 목록을 조회 한다.")
+    @Test
+    void getLines() {
+        // given
+        Long station1 = 지하철_역_생성("강남역").jsonPath().getLong("id");
+        Long station2 = 지하철_역_생성("양재역").jsonPath().getLong("id");
+        Long station3 = 지하철_역_생성("역삼역").jsonPath().getLong("id");
+        Long station4 = 지하철_역_생성("선릉역").jsonPath().getLong("id");
```

Comment:
```suggestion
        Long 강남역 = 지하철_역_생성("강남역").jsonPath().getLong("id");
        Long 양재역 = 지하철_역_생성("양재역").jsonPath().getLong("id");
        Long 역삼역 = 지하철_역_생성("역삼역").jsonPath().getLong("id");
        Long 선릉역 = 지하철_역_생성("선릉역").jsonPath().getLong("id");
```

저는 인수테스트에서 한글로 쓰는게 더 효율적이라고 생각이 들더군요~  API를 사용하는 사람들을 위한 배려를 위한 부분도 있기 때문이에요.